src/main/java/com/mraof/minestuck/player/Caegar.java

```diff
@@ -0,0 +1,5 @@
+package com.mraof.minestuck.player;
+
+public class Caegar
+{
+}
```

Comment:
the whole class?

Reply:
Yup, nowhere in the code does it appear used. In IntelliJ it will show up as grey if its not used, but you can also right click the text and press Find Usages to see if it appears elsewhere